app/src/main/java/lt/vilnius/tvarkau/NewProblemActivity.java

```diff
@@ -110,4 +152,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    private void showToast(String text){
```

Comment:
Well, let's better inline this method instead. It's used only once. If you'd put it where it's called, then there would be no need to check, what this method does.